src/Button/Button.js

```diff
@@ -26,6 +26,8 @@ type Props = {
   circular?: boolean,
   /** Disables the Button */
   disabled?: boolean,
+  /** Element to be used as the root node - e.g. 'a' */
```

Comment:
If we are not going to have a separate example that demonstrates use of the `element` prop to create a link that is styled like a button, then we should add a bit more description here.

```
/** Element to be used as the root node - e.g. 'a' can be used to create a link that is styled like a button. */
```